scripts/generate_scripts_functions.py

```diff
@@ -214,10 +214,10 @@ def arg_type(x, el):
     if 'AllowGlobalVar' in el.attrib:
         allowGlobal = True if el.attrib['AllowGlobalVar'] == 'true' else False
     if allowLocal == False and allowGlobal == True:
-        outRef = 'Global'
+        outRef = '&'
         outName = 'arg' + str(x) + 'G'
     elif allowLocal == True and allowGlobal == False:
-        outRef = 'Local'
+        outRef = '&'
```

Comment:
The variable name will contain the fact it's a local or a global for un-named variables. I'm not sure if we ever need to know if a value came from a local or a global though.